tests/logging-utils/test_compat_with_typing_Generic.py

```diff
@@ -0,0 +1,16 @@
+import logging
+from typing import GenericMeta, TypeVar
+
+from eth_utils import HasLoggerMeta
+
+T = TypeVar("T")
```

Comment:
~~Maybe this module is missing some test for the class still being generic, like the one in `tests/logging-utils/test_compat_with_abc_ABC.py`?~~

Nevermind, I see it in `tests/type-checks/mypy_typing_of_has_logger.py`. Maybe remove this `TypeVar` and add a comment that the `Generic` testing is in that other test file?